ee/tabby-db/src/job_runs.rs

```diff
@@ -96,6 +113,31 @@ impl DbConn {
         Ok(job_runs)
     }
 
+    pub async fn compute_job_stats(&self, jobs: Option<Vec<String>>) -> Result<JobStatsDAO> {
+        let condition = match jobs {
+            Some(jobs) => {
+                let jobs: Vec<_> = jobs.into_iter().map(|s| format!("{s:?}")).collect();
+                let jobs = jobs.join(", ");
+                format!("AND job IN ({jobs})")
+            }
+            None => "".into(),
+        };
+
+        let cutoff = Utc::now() - Duration::days(7);
+
+        let stats = sqlx::query_as(&format!(
+            r#"SELECT
+                SUM(exit_code == 0) AS success,
+                SUM(exit_code != 0 AND exit_code IS NOT NULL) AS failed,
+                SUM(exit_code IS NULL) AS pending FROM job_runs
+                WHERE start_ts > ? {condition};"#
```

Comment:
```suggestion
                WHERE created_at > ? {condition};"#
```

column `start_ts` is a mistake, good to remove in future refactors.

Reply:
Any reason we don't remove it?

Reply:
it's just better to do it in a separate refactor PR. can be removed anytime